src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -202,9 +240,13 @@
  *   </tr>
  *   </tbody>
  * </table>
+ * <p> The virtual thread scheduler can be monitored and managed with the
+ * {@link jdk.management/jdk.management.VirtualThreadSchedulerMXBean} management
```

Comment:
Links to jdk. modules will break the Java SE API doc from `make docs-javase` when it runs the jdkCheckLinks test. We may have to use `{@code}` instead.